Directory.Packages.props

```diff
@@ -60,26 +60,26 @@
   <!-- non-prod packages -->
   <!-- 'net7.0' is the default `TargetFramework`. Use `VersionOverride` in the project to override the package versions from a different `TargetFramework` -->
   <ItemGroup>
-    <PackageVersion Include="BenchmarkDotNet" Version="[0.13.5,0.14)" />
+    <PackageVersion Include="BenchmarkDotNet" Version="[0.13.6,0.14)" />
     <PackageVersion Include="CommandLineParser" Version="[2.9.1,3.0)" />
-    <PackageVersion Include="Grpc.AspNetCore" Version="[2.50.0,3.0)" />
-    <PackageVersion Include="Grpc.AspNetCore.Server" Version="[2.48.0, 3.0)" />
-    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.1" />
+    <PackageVersion Include="Grpc.AspNetCore" Version="[2.55.0,3.0)" />
+    <PackageVersion Include="Grpc.AspNetCore.Server" Version="[2.55.0, 3.0)" />
+    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="7.0.9" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.9" />
```

Comment:
Not sure if we could just use the single version here https://github.com/open-telemetry/opentelemetry-dotnet/blob/a76564e33bc69e796f672544c0d0e23090dc587c/test/OpenTelemetry.Instrumentation.AspNetCore.Tests/OpenTelemetry.Instrumentation.AspNetCore.Tests.csproj#L33-L39. (can be checked outside of this PR)